src/knex-store.ts

```diff
@@ -166,6 +166,17 @@ function knex_store(this: any, options: Options) {
     }
   )
 
+  seneca.message(
+    'sys:entity,transaction:adopt',
+    async function (msg: any, reply: any) {
+      const transaction = msg.details()
```

Comment:
this should be get_handle - see https://github.com/senecajs/seneca-entity/blob/master/src/entity.ts